python/llama-index-server/llama_index/server/models/artifacts.py

```
@@ -5,6 +5,7 @@
 from llama_index.core.workflow.events import Event
 from llama_index.server.models.chat import ChatAPIMessage
 from pydantic import BaseModel
+from llama_index.server.utils.inline import get_inline_annotations
```

Pylint (3.3.7)

[error] Unable to import 'llama_index.server.utils.inline'

(E0401)